File: Python/figury.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  figury.py
#  
# Program buduje wypełniony prostokąt o bokach podanych przez 
# użytkownika za pomocą podanego znaku  

def prostokat1(a, b, znak):
    
    a = int(input("Pierwszy bok: "))
    b = int(input("Drugi bok: "))
    c = int(input("Znak wypełnienia: "))

    for i in range(a):
        for j in range(b):
            print(c, end='')
        print()
        
def prostokat2(a, b, znak):
    
    for i in range(a):
        for j in range(b):
            if i 
            if j == 0 or j == b - 1:
                print(c, end='')
            else:
                print(" ", end='')
        print()


def main(args):
    
    a = int(input("Pierwsza liczba: "))
    b = int(input("Druga liczba: "))
    
    for i in range(a):
        for j in range(b):
            print('#', end='')
        print()
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/figury.cpp
/*
 * figury.cpp
 * 
 */


#include <iostream>

using namespace std;

void prostokat2(int x, int y, char znak) {
    for (int i = 1; i <= x; i++) {
        for (int j = 1; j <= y; j++) {
            if (i != j) {
                cout << i << j << " ";
                ile++;
            }
        }
        cout << endl;
    }
}


int main(int argc, char **argv)
{
	int a, b; // deklaracja
    a = b = 0; // inicjacja
    // int a = 0; // definicja
    cout << "Podaj bok pierwszy: ";
    cin >> a;
    cout << "Podaj bok drugi: ";
    cin >> b;
    
    char znak;
    cout << "Podaj znak: ";
    cin >> znak;
    
    prostokat2(a, b, znak);
	return 0;
}

